static/js/containers/auth/LoginPage.js

```diff
@@ -29,32 +29,48 @@ import ExternalLogins from "../../components/ExternalLogins"
 
 type LoginPageProps = {
   history: Object,
-  formError: ?string
+  formError: ?string,
+  clearEndpointState: Function
 } & WithFormProps<EmailForm>
 
-export const LoginPage = ({ renderForm, formError }: LoginPageProps) => (
-  <div className="content auth-page login-page">
-    <div className="main-content">
-      <Card className="login-card">
-        <h3>Log In</h3>
-        <MetaTags>
-          <title>{formatTitle("Log In")}</title>
-        </MetaTags>
-        {renderForm({ formError })}
-        <ExternalLogins />
-        <div className="alternate-auth-link">
-          Not a member? <Link to={REGISTER_URL}>Sign up &gt;</Link>
+export class LoginPage extends React.Component<LoginPageProps> {
+  componentWillUnmount() {
+    const { formError, clearEndpointState } = this.props
+    if (formError) {
+      clearEndpointState()
+    }
+  }
+
+  render() {
+    const { renderForm, formError } = this.props
+
+    return (
+      <div className="content auth-page login-page">
+        <div className="main-content">
+          <Card className="login-card">
+            <h3>Log In</h3>
+            <MetaTags>
+              <title>{formatTitle("Log In")}</title>
+            </MetaTags>
+            {renderForm({ formError })}
+            <ExternalLogins />
+            <div className="alternate-auth-link">
+              Not a member? <Link to={REGISTER_URL}>Sign up &gt;</Link>
```

Comment:
Since you're touching this line anyway, can you remove the &gt; so it's the same as https://docs.google.com/document/d/1_sD0-ng7PsYmvmmOjoTYcguygLVctx2RZqUbJsDPvR8/edit#heading=h.45j4wcfi4l54